client/src/client.rs

```diff
@@ -1011,4 +1068,64 @@ mod tests {
         let expected_value = format!("Basic {}", ENCRYPTED_CREDENTIALS);
         assert_eq!(value, &expected_value);
     }
+
+    mod query_errors_handling {
```

Comment:
```suggestion
    #[cfg(test)]
    mod query_errors_handling {
```

Reply:
It looks unnecessary, but ok. `cargo test` takes it anyway